test/framework/integration_framework/iroha_instance.cpp

```diff
@@ -52,17 +56,29 @@ namespace integration_framework {
     instance_->storage->insertBlock(block);
   }
 
+  void IrohaInstance::setMstGossipParams(
+      std::chrono::milliseconds mst_gossip_emitting_period,
+      uint32_t mst_gossip_amount_per_once) {
+    BOOST_ASSERT_MSG(
+        !instance_,
+        "Gossip propagation params must be set before Irohad is started!");
+    opt_mst_gossip_params_ = iroha::GossipPropagationStrategyParams{
+        .period = mst_gossip_emitting_period,
+        .amount = mst_gossip_amount_per_once};
```

Comment:
That's from C90, not C++ :(
[It seems](https://en.cppreference.com/w/cpp/language/aggregate_initialization) it going to be in 20th standard only